cmd2/argparse_custom.py

```diff
@@ -1033,7 +1033,7 @@ def _ArgumentParser_check_value(self: argparse.ArgumentParser, action: argparse.
 ############################################################################################################
 
 # noinspection PyPep8Naming,PyProtectedMember
-def _SubParsersAction_remove_parser(self: argparse._SubParsersAction, name: str) -> None:
+def _SubParsersAction_remove_parser(self: Any, name: str) -> None:
```

Comment:
`self` has to be an `argparse._SubParsersAction` since this is a custom method that we add to the `_SubParsersAction` class.

Reply:
Yeah I'll just add `# type: ignore` instead.  Thanks